maps/away/ships/idris/idris_cruiser.dm

```diff
@@ -0,0 +1,102 @@
+/datum/map_template/ruin/away_site/idris_cruiser
+	name = "Idris Cruiser"
+	description = "A small luxury cruiser run by Idris Incorporated's subsidiary, Celestial Cruises. The Argentum-class is more of a yacht than a proper cruise ship, and is easily dwarfed by the fleet's larger vessels. However, it makes up for its diminuitive size by its speed, flexibility, and low maintenance cost. It adopts a unique wandering business model, where it roams the Spur and caters to tired traveling vessel crews seeking a getaway among the stars. It comes with a bar and restaurant, a pool, a spa, and a viewing lounge, as well as four suites for overnight stayers."
+	suffixes = list("ships/idris/idris_cruiser.dmm")
+	sectors = list(ALL_CORPORATE_SECTORS)
```

Comment:
I feel like this includes too many sectors.

Like SECTOR_WEEPING_STARS for example, which I feel like wouldn't really be a great place for a luxury cruiser to get rich passengers to use its facilities.